packages/grid/x-data-grid/src/components/panel/filterPanel/GridFilterInputBoolean.tsx

```diff
@@ -27,42 +31,43 @@ export function GridFilterInputBoolean(props: GridFilterInputValueProps & TextFi
   }, [item.value]);
 
   return (
-    <rootProps.slots.baseTextField
-      // TODO: use baseSelect slot
-      label={apiRef.current.getLocaleText('filterPanelInputLabel')}
-      value={filterValueState}
-      onChange={onFilterChange}
-      select
-      variant="standard"
-      SelectProps={{
-        native: isSelectNative,
-        displayEmpty: true,
-        ...rootProps.slotProps?.baseSelect,
-      }}
-      InputLabelProps={{
-        shrink: true,
-      }}
-      inputRef={focusElementRef}
-      {...others}
-      {...rootProps.slotProps?.baseTextField}
-    >
-      <rootProps.slots.baseSelectOption {...baseSelectOptionProps} native={isSelectNative} value="">
-        {apiRef.current.getLocaleText('filterValueAny')}
-      </rootProps.slots.baseSelectOption>
-      <rootProps.slots.baseSelectOption
-        {...baseSelectOptionProps}
-        native={isSelectNative}
-        value="true"
-      >
-        {apiRef.current.getLocaleText('filterValueTrue')}
-      </rootProps.slots.baseSelectOption>
-      <rootProps.slots.baseSelectOption
-        {...baseSelectOptionProps}
+    <rootProps.slots.baseFormControl {...rootProps.slotProps?.baseFormControl} variant="standard">
```

Comment:
Do we need a `FormControl` with this component. 🤔 
If yes, then do you think it makes sense to add to other `GridFIlterInputxxx` items too?

From [documentation](https://mui.com/material-ui/api/form-control/#import), I felt it is more suitable in a form when we have multiple fields under the `FormControl` component which access the same context for variant etc.
> Provides context such as filled/focused/error/required for form inputs. Relying on the context provides high flexibility and ensures that the state always stays consistent across the children of the FormControl.

Reply:
Indeed, it seems to be unnecessary.
I was following https://mui.com/material-ui/react-select/#basic-select to add label, but it works fine without FormControl